pyiron_atomistics/sphinx/base.py

```diff
@@ -1940,7 +1940,7 @@ def collect_spins_dat(self, file_name="spins.dat", cwd=None):
             return None
         spins = np.loadtxt(posixpath.join(cwd, file_name))
         self._parse_dict["atom_scf_spins"] = self.splitter(
-            np.array([ss[self._job.id_spx_to_pyi] for ss in spins[:, 1:]]),
+            [ss[self._job.id_spx_to_pyi] for ss in spins[:, 1:]],
```

Comment:
Why do you remove the `numpy.array()` ?

Reply:
Oh sorry it was not meant to be part of this PR.

Reply:
```suggestion
            np.array([ss[self._job.id_spx_to_pyi] for ss in spins[:, 1:]]),
```